### DISCUSSION TITLE
The ability to set all columns hideable for a datagrid

### DISCUSSION THREAD

**Discussion by KingMario** [vmware-clarity/ng-clarity#776] [General]
Currently, one has to use the `clrDgHideableColumn` directive to set a column hideable.

Please consider adding a boolean property for the datagrid component, when set to true, allows all the columns to be hideable without specifically applying `clrDgHideableColumn` to every column.

Thanks~~~

**Comment by kevinbuhmann** [comment]
The datagrid requires the `*clrDgHideableColumn` structural directive for each hideable column so that we can grab the `TemplateRef` for the column title. We use that `TemplateRef` to render the column title in the "Manage Columns" popup.

Due this this, we cannot add a single boolean property to make all columns hideable.

**Reply by KingMario** [reply]
Got it. Thanks
